File: level_2/narrative5_average_price/langs/compiler.cpp
int main() 
{ 
    int n;
    int sum = 0;
    cin >> n;
    int arr[n];
    
    for(int i=0;i<n;++i)
        cin>>arr[i];
    
    cout<<average_price(n, arr);
    
    return 0;
}

File: level_2/frequent_integer_not_blocked/langs/compiler.java
public static void main(String[] args)
{
    Scanner scanner = new Scanner(System.in);

    int n = scanner.nextInt();
    int[] arr = new int[n];

    for (int i = 0; i < n; i++)
        arr[i] = scanner.nextInt();
            
    System.out.println(averagePrice(n, arr));
}